pkg/cmd/server/start/network_args.go

```diff
@@ -16,6 +16,13 @@ type NetworkArgs struct {
 	// ServiceNetworkCIDR is the CIDR string representing the network that service IP
 	// addresses will be allocated from
 	ServiceNetworkCIDR string
+	// IngressIPNetworkCIDR controls the range to assign ingress ips
+	// from for services of type LoadBalancer on bare metal. If empty,
+	// ingress ips will not be assigned. It may contain a single CIDR
+	// that will be allocated from. For security reasons, you should
+	// ensure that this range does not overlap with the CIDRs reserved
+	// for external ips, nodes, pods, or services.
+	IngressIPNetworkCIDR string
```

Comment:
You don't need to add this here - we stopped allowing all but "critical" new flags here.  Just adding it to the config is sufficient